include/gz/rendering/Utils.hh

```diff
@@ -97,6 +97,19 @@ namespace gz
     gz::math::AxisAlignedBox transformAxisAlignedBox(
         const gz::math::AxisAlignedBox &_box,
         const gz::math::Pose3d &_pose);
+
+    /// \brief Get the camera's intrinsic matrix.
```

Comment:
Since this is not part of a class anymore, it would be better to say convert a given camera projection matrix to an intrinsic matrix. Also reference to `ProjectionMatrix` should be `Camera::ProjectionMatrix`

Reply:
Done https://github.com/gazebosim/gz-rendering/pull/755/commits/ac8432148668f7f9fe65acdba2f05ddd5232af8a